[2025-02-04T22:05:59.624+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T22:05:59.638+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-04T22:05:57.719723+00:00 [queued]>
[2025-02-04T22:05:59.646+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-04T22:05:57.719723+00:00 [queued]>
[2025-02-04T22:05:59.647+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-04T22:05:59.665+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-04 22:05:57.719723+00:00
[2025-02-04T22:05:59.672+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=94) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T22:05:59.675+0000] {standard_task_runner.py:72} INFO - Started process 96 to run task
[2025-02-04T22:05:59.674+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'extract_data', 'manual__2025-02-04T22:05:57.719723+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmpx9k1_dr3']
[2025-02-04T22:05:59.835+0000] {standard_task_runner.py:105} INFO - Job 60: Subtask extract_data
[2025-02-04T22:05:59.862+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-04T22:05:57.719723+00:00 [running]> on host d79180ab324d
[2025-02-04T22:05:59.905+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T22:05:57.719723+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T22:05:57.719723+00:00'
[2025-02-04T22:05:59.906+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T22:06:00.393+0000] {dag_que_faire_a_paris.py:40} INFO - âœ… Extraction completed successfully!
[2025-02-04T22:06:00.395+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-04T22:06:00.401+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T22:06:00.402+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=extract_data, run_id=manual__2025-02-04T22:05:57.719723+00:00, execution_date=20250204T220557, start_date=20250204T220559, end_date=20250204T220600
[2025-02-04T22:06:00.453+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T22:06:00.631+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-04T22:06:00.636+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
